### ISSUE TITLE
Passing mean and std to normalization transform. 

### ISSUE THREAD

**Issue by farazahmeds** [fepegar/torchio#228]
So I am using pretrained model the `mean` and `std `values are provided as such:

` mean = [0.43216, 0.394666, 0.37645] and std = [0.22803, 0.22145, 0.216989]`

How can I pass them to the normalization transform, ` torchio.transforms.preprocessing.ZNormalization` ? 

Thankyou.

**Comment by fepegar** [comment]
TorchIO is not optimized for 2D RGB images, and that transform for now only does zero mean and unit variance. It might be easier for you to just write a new transform. Can you share some code and what you're trying to achieve?

**Comment by farazahmeds** [comment]
> TorchIO is not optimized for 2D RGB images, and that transform for now only does zero mean and unit variance. It might be easier for you to just write a new transform. Can you share some code and what you're trying to achieve?

Sorry I didn't clarify my data is single channel, Nifty volumes, I am trying to use the [pretrained video classification model (ResNet3D)](https://pytorch.org/docs/stable/torchvision/models.html#resnet-3d)

better question maybe: 

how can I use their recommended mean and std values for single channel to normalize my data?

**Comment by fepegar** [comment]
As far as I know, that model has been trained on video clips with shape `(3, L, 112, 112)` where `L` is the number of frames. Videos are very different from medical images, I really don't think you should normalize them using that transform.

I coincidentally happen to also be working with videos at the moment. I suggest you take a look at all the files in this folder of the `torchvision` repo: https://github.com/pytorch/vision/tree/master/references/video_classification

You'll find, amongst many other useful resources, the transform you're looking for:
https://github.com/pytorch/vision/blob/0344603ece20ba5e4086aa8c49b2cce0829aefe6/references/video_classification/transforms.py#L97-L103
